src/recon_test/CMakeLists.txt

```diff
@@ -32,13 +41,83 @@ set(${dir_INVOLVED_TEST_EXE_SOURCES}
+                        "ece24f080d81b32679e3ae95361d41dd")
+        # 3
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_data_test3.root"
+                    "${ROOT_DATA_URL}/root_data_test3.root?download=1"
+                    "3d6d741d0709d8f0e034b63fba24eaa1")
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_header_test3.hroot"
+                    "${ROOT_DATA_URL}/root_header_test3.hroot?download=1"
+                    "9d4aa131344e887fbb15cd1004a8255e")
+        # 4
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_data_test4.root"
+                        "${ROOT_DATA_URL}/root_data_test4.root?download=1"
+                        "c07d50333dbb2f42a7df6f3cc8c060ac")
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_header_test4.hroot"
+                        "${ROOT_DATA_URL}/root_header_test4.hroot?download=1"
+                        "4533e5d010abe190704546f79e1b5b8d")
+        # 5
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_data_test5.root"
+                        "${ROOT_DATA_URL}/root_data_test5.root?download=1"
+                        "178955075c3d46cebd62b68c8b807643")
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_header_test5.hroot"
+                        "${ROOT_DATA_URL}/root_header_test5.hroot?download=1"
+                        "af8d2fe7a0a4668aaed2b2977096629c")
+        # 6
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_data_test6.root"
+                        "${ROOT_DATA_URL}/root_data_test6.root?download=1"
+                        "e57b6668fa2cabf393523db85d0a994f")
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_header_test6.hroot"
+                        "${ROOT_DATA_URL}/root_header_test6.hroot?download=1"
+                        "6f6025593ef7891dbc4fa2a4964ad1f8")
+        # 7
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_data_test7.root"
+                        "${ROOT_DATA_URL}/root_data_test7.root?download=1"
+                        "7b3faa81b3fe35f44e577cbab5b6a213")
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_header_test7.hroot"
+                        "${ROOT_DATA_URL}/root_header_test7.hroot?download=1"
+                        "174b9dfa3d88f9bbcaebc4c56e1ebf93")
+        # 8
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_data_test8.root"
+                        "${ROOT_DATA_URL}/root_data_test8.root?download=1"
+                        "24150ce5d655212dc975989b3dbe4b33")
+        get_zenodo_file("${ROOT_STIR_consistency_data_dir}/root_header_test8.hroot"
+                        "${ROOT_DATA_URL}/root_header_test8.hroot?download=1"
+                        "a821a3eac0000544f000cd5c74150023")
+    endif()
+
+    # Add test if the data is downloaded (only checking root_header_test1.hroot)
+    if(EXISTS "${ROOT_STIR_consistency_data_dir}/root_header_test1.hroot")
```

Comment:
are you sure this works ok? i.e. is the `if(EXISTS)` evaluated *after* the download or before? I guess you'll need to check if this test is run once you deleted the data.

Reply:
Configuring downloads the data and then checks if `root_header_test1.hroot` exists. Then it adds the test to the list of tests. 

If you download, configure and build, then delete the data, the test will be run and fail.